src/main/java/com/github/invictum/mei/CustomPlugin.java

```diff
@@ -0,0 +1,27 @@
+package com.github.invictum.mei;
+
+import org.bukkit.plugin.java.JavaPlugin;
+
+public abstract class CustomPlugin extends JavaPlugin {
+    private Backend backend = null;
+
+    public Backend getBackend() {
+        if (backend == null) {
+            this.reloadBackend();
+        }
+        return backend;
+    }
+
```

Comment:
make it as private, remove initialization. Other thinks will be done regarding [comment](https://github.com/Invictum/minecraft-executor-interface/pull/1#discussion_r108872078)